R/rescale_catchments.R

```diff
@@ -150,6 +183,8 @@ rescale_catchment_characteristics <- function(vars, lookup_table,
     # download characteristics for the requested comids
     catchment_characteristics <- get_catchment_characteristics(varname = vars$characteristic_id,
                                               ids = unique(lookup_table$comid))
+    # omit any duplicated rows in the returned attribute table
+    catchment_characteristics <- distinct(catchment_characteristics)
```

Comment:
Actually already doing this here: https://github.com/DOI-USGS/nhdplusTools/blob/hydroloom/R/get_vaa.R#L301